src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -19,15 +22,24 @@ public void canGet() {
 
   @Test
   public void canPutAndGet() {
-    String rule = "priority: t, s, c, b, a, m, g\nfallback-policy: l no-circulation r no-hold n basic-notice o basic-overdue i basic-lost-item\n";
-
-    circulationRulesFixture.updateCirculationRules(rule);
+    UUID lp1 = UUID.randomUUID();
+    UUID lp2 = UUID.randomUUID();
+    UUID rp1 = UUID.randomUUID();
+    UUID rp2 = UUID.randomUUID();
+    UUID np1 = UUID.randomUUID();
+    UUID np2 = UUID.randomUUID();
+    UUID op1 = UUID.randomUUID();
+    UUID op2 = UUID.randomUUID();
+    UUID ip1 = UUID.randomUUID();
+    UUID ip2 = UUID.randomUUID();
+
+    String rule = String.format("priority: t, s, c, b, a, m, g\nfallback-policy: l %s r %s n %s o %s i %s \n", lp1, rp1, np1, op1, ip1);
+    setRules(rule);
 
     assertThat(getRulesText(), is(rule));
 
-    rule = "priority: t, s, c, b, a, m, g\nfallback-policy: l loan-forever r two-week-hold n two-week-notice o forever-overdue i forever-lost-item\n";
-
-    circulationRulesFixture.updateCirculationRules(rule);
+    rule = String.format("priority: t, s, c, b, a, m, g\nfallback-policy: l %s r %s n %s o %s i %s \n", lp2, rp2, np2, op2, ip2);
+    setRules(rule);
```

Comment:
The string template for the rule appears to be the same in both cases, could it be a constant template?